module-2/backend/week_three.md

```diff
@@ -10,14 +10,45 @@ Note: When you're done, submit a PR with a reflection in the comments about how
 ### Questions
 
 1. What is the entry at the command line to create a new rails app?
+rails new project_name_here"
+
 2. What do Models generally inherit from in rails?
+active record
+
 3. What do Controllers generally inherit from in a rails project?
+application controller
+
 4. How would I create a route if I wanted to see a specific horse in my routes fitle assuming I'm sticking to standard conventions and that I didn't want other CRUD functionality?
+get 'horses' => 'horse#show' OR resources :horses, only: [:show]
+
 5. What rake task is useful when looking at routes, and what information does it give you?
+rake routes tells you all the wonderful routes you have available to you
+
 6. What is an example of a route helper? When would you use them?
+nested routes. When things can't exist without other things it would be useful to have nested routes
+
 7. What's the difference between what `_url` and `_path` return when combined with a routes prefix?
+I always know when I see the error, but after Job Tracker all weekend I can't for the life of me think of why
+
 8. What are strong params and why are the necessary?
+it is a private method in your controller that makes it so people can't alter the html and params when forms are submitted.
+
 9. What role does `form_for` play in helping us create our forms?
+it creates a form for a specific object i.e. @comments, @contacts, @jobs, etc
+
 10. How does `form_for` know where to submit the user's input?
+In the controller
```

Comment:
`form_for` makes this decision based off of the object that it's passed (e.g. `@comments` in your example above), and whether or not it's been saved to the database. So, if `@comments` had not been saved, it would send a POST request to `/comments`. If it *had* been saved (i.e. it was an existing comment), it would send a PATCH request to `/comments/:id` (all assuming no nested or namespaced routes)